### DISCUSSION TITLE
What are your preferred chorus effect settings ?

### DISCUSSION THREAD

**Discussion by jjazzboss** [FluidSynth/fluidsynth#1189] [General]
If you use FluidSynth to render Midi files or drums/bass/keyboard/..., could you share your preferred chorus effect settings ? (nr, speed, depth, level).

_Background:_ 
I'm working to add FluidSynth in my app. For simplicity I'd like to propose users only 2-3 presets for reverb and chorus. It's not too difficult for reverb (e.g. from small room to large hall), but I find it more difficult to identify 2 or 3 "different yet usable" presets for chorus.

Thanks!

**Comment by derselbst** [comment]
I usually prefer to keep the `depth` and `speed` params at their default and then I start playing around with `nr` and `level` only. I don't have a preferred value for them though - it depends on the tune. I never use triangular modulation.

**Comment by jjazzboss** [comment]
Thanks!
